app/utils.py

```diff
@@ -954,6 +955,24 @@ def should_show_inline_photo(self, photo_url):
         else:
             return True
 
+    URL_PARSE_QUERY_INDEX = 4;
```

Comment:
Remove the semicolon.

You can optionally move it to inside get_thumbnail_url() unless you plan to use it in other methods.